### PR TITLE
Use lowercase file names `xcat-core.repo` and `xcat-dep.repo` in the download page.

### PR DESCRIPTION
**Pull request by neo954**
`xCAT-core.repo` -> `xcat-core.repo`
`xCAT-dep.repo` -> `xcat-dep.repo`

### CODE REVIEW

File: download.html [hunk 1]
```diff
@@ -14,7 +14,8 @@
 <a href="http://xcat-docs.readthedocs.io">Documentation</a> |
 <a href="https://sourceforge.net/p/xcat/mailman/">Mailling List</a> |
 <a href="https://github.com/xcat2/xcat-core/issues">Issues</a> |
-<a href="https://github.com/xcat2/xcat-core">GitHub</a>
+<a href="https://github.com/xcat2/xcat-core">GitHub</a> |
+<a href="http://webchat.freenode.net/?channels=xcat">IRC</a>
```
Comment (whowutwut) [inline download.html]: Nice catch!

**Comment by whowutwut** [comment]
@neo954 I think we need some changes in the https://github.com/xcat2/xcat.org/blob/master/promote_build.py when it moves the repo files around.. still referencing xCAT-core.repo

**Comment by whowutwut** [comment]
May also need to use this PR to address #88

**Comment by neo954** [comment]
Hello @vhu,

You are right, we need to update these places after 2.14 GA. This pull request is updated.

**Comment by hu-weihua** [comment]
It is ok for me, let's us merge it to update ``xcat.org`` early. thanks @neo954

**Pull request merged**